src/grt/Makefile.inc

```diff
@@ -56,7 +56,7 @@ else
     GRT_EXTRA_LIB=-ldl -lm $(GRT_ELF_OPTS)
   endif
   ifeq ($(filter-out netbsd freebsd% dragonfly%,$(osys)),)
-    GRT_EXTRA_LIB=-lm $(GRT_ELF_OPTS)
+    GRT_EXTRA_LIB=-lm
```

Comment:
If you get ride of GRT_ELF_OPTS, you might have issues with VPI.
I am ok with that change, but it must affect only freebsd.